src/Apps/Settings/Routes/MyCollection/Components/DownloadAppBadgesDark.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Flex, Image, Spacer } from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+import {
+  useDeviceDetection,
+  Device,
+  DOWNLOAD_APP_URLS,
+} from "Utils/Hooks/useDeviceDetection"
+
+export const DownloadAppBadgesDark: React.FC = () => {
+  const { device } = useDeviceDetection()
+
+  if (device === Device.Android) {
+    return <PlayStoreBadge />
+  }
+
+  if (device === Device.iPhone) {
+    return <AppStoreBadge />
+  }
+
+  return (
+    <Flex>
+      <AppStoreBadge />
+
+      <Spacer ml={2} />
+
+      <PlayStoreBadge />
+    </Flex>
+  )
+}
+
+const PlayStoreBadge: React.FC = () => {
+  return (
+    <RouterLink to={DOWNLOAD_APP_URLS[Device.Android]} target="_blank">
+      <Box width={136} height={40}>
```

Comment:
I updated the width for the icon here to match the one inside [DownloadAppBadgees](https://github.com/artsy/force/blob/main/src/Components/DownloadAppBadges/DownloadAppBadge.tsx#L78)